vnext/Microsoft.ReactNative/Views/VirtualTextViewManager.h

```diff
@@ -16,6 +16,19 @@ struct VirtualTextShadowNode final : public ShadowNodeBase {
   TransformableText transformableText{};
 
   void AddView(ShadowNode &child, int64_t index) override;
+
+  bool m_isHighlighted = false;
+  folly::dynamic m_ColorValue = nullptr;
+  size_t m_length = 0;
+
+  typedef struct HighlightData {
+    std::vector<std::unique_ptr<HighlightData>> data;
+    size_t spanIdx = 0;
+    bool isHighlighted = false;
+    folly::dynamic color;
+  } HighlightData;
+
```

Comment:
no need for typedef struct X { } X in C++

Reply:
oh yes, I don't know why I wrote that :D